src/tpm/kmyth_seal_unseal_impl.c

```diff
@@ -151,10 +154,82 @@ int tpm2_kmyth_seal(uint8_t * input,
     return 1;
   }
 
+  // optional argument allowing user to receive a hex dump of the policy digest,
+  // to be used to calculate a second policy for policyOR authorization
+  if (bool_trial_only == 1)
+  {
+    // size of string is 128 chars for the size of the TPM2B_DIGEST buffer + 4
```

Comment:
I think this is wrong. The size of the TPM2B_DIGEST struct is two bytes for its 'size' member and for the current kmyth parameters, it is 32 bytes for the 'buffer' member of that struct. 

I believe that we should print out only the contents of the TPM2B_DIGEST struct, which should require 64 hex characters (or, more portably, twice the value contained in the 'size' member).

Currently, this prints out 4 hex characters for the size concatenated with 64 hex characters for the digest (hash result) padded with 64 zero hex characters.

Reply:
Addressed in subsequent commits.